src/handlers/comment.ts

```diff
@@ -1,6 +1,28 @@
 import { addCommentToIssue, isUserAdminOrBillingManager } from "../shared/issue";
 import { Context } from "../types/context";
 import { isCommentEvent } from "../types/typeguards";
+import commandParser, { AllowedCommand, CommandArguments, isValidCommand } from "./command-parser";
+
+const commandHandlers: { [k in AllowedCommand]: (context: Context, commandArguments: CommandArguments) => Promise<void> } = {
+  async allow(context, { username, labels }: CommandArguments) {
+    const logger = context.logger;
+    if (!isCommentEvent(context)) {
+      return logger.debug("Not an comment event");
```

Comment:
I guess supabase is just for the logger, but I thought we have a module in our sdk for this. 

It really doesn't make sense to enable supabase support manually on all of our plugins just for logs. It should be abstracted away in an SDK

Reply:
There is a package for this that had to be reworked to be usable within the new plugins (was too much coupled with the bot v1) and there is a PR for that:
https://github.com/ubiquity/ubiquibot-logger/pull/4